doc/advanced_configuration.rst

```diff
@@ -148,19 +148,23 @@ provided the relative paths to `conf.py` of each sub gallery folder.
 Sorting gallery examples
 ========================
 
-Within a given gallery (sub)section, the example files by default are ordered
-by the amount of code, i.e. :class:`sphinx_gallery.sorting.AmountOfCodeSortKey`
-as::
+Within a given gallery (sub)section, the example files are ordered by
+using the standard :func:`sorted` function with the ``key`` argument by default
+set to
+:class:`NumberOfCodeLinesSortKey(src_dir) <sphinx_gallery.sorting.NumberOfCodeLinesSortKey>`,
+which sorts the files based on the number of code lines::
 
-    from sphinx_gallery.sorting import AmountOfCodeSortKey
+    from sphinx_gallery.sorting import NumberOfCodeLinesSortKey
     sphinx_gallery_conf = {
         ...
-        'within_subsection_order': AmountOfCodeSortKey,
+        'within_subsection_order': NumberOfCodeLinesSortKey,
     }
 
-However, other options exist that can be instantiated for use with
+However, multiple convenience classes are provided for use with
```

Comment:
If you're going to make edits can you make this "In addition" instead of "However"?

If there aren't any more commits needed then I can edit and then merge (but see my other comment)